pkg/util/ftp/auth.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ftp
```

Comment:
Better to move it to a standalone package: util/auth. I feel it won't be easy for the developers to find the right util when it's under ftp. cc @antoninbas

Reply:
Agreed.  moved.